airflow/bin/cli.py

```diff
@@ -2032,6 +2047,11 @@ class CLIFactory(object):
         'task_params': Arg(
             ("-tp", "--task_params"),
             help="Sends a JSON params dict to the task"),
+        'post_mortem': Arg(
+            ("-pm", "--post_mortem"),
+            action="store_true",
```

Comment:
Seems the default value here would be `True`, i.e. the post_mortem will be invoked by default. But this is different from the previous behavior. Personally I would suggest to use `False` as default value here, to make behavior consistent.

Reply:
In argparse, store_true sets the default to false, so I think this is correct.

Reply:
Yes you're right. Thanks for clarification.